lib/iex/lib/iex/autocomplete.ex

```diff
@@ -229,6 +229,9 @@ defmodule IEx.Autocomplete do
       %{kind: :module, type: :elixir, name: Enum.at(parts, depth - 1)}
     end
     |> Enum.uniq
+    |> Enum.filter(fn(%{name: name}) ->
+      !match?(":" <> _, inspect(String.to_atom("Elixir." <> name)))
```

Comment:
`String.starts_with?/2` instead of `match?`. Use `not` instead of `!` for boolean values (booleans are guaranteed for functions ending with `?`.